docker/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:18-buster
```

Comment:
I  dont understand much about Docker. Atlas project uses node version equals to 12. Is there a problem?

Reply:
Looking the image node version at the docker hub we can see the latest version released is the `18`, but since they are given much support to it, I don't think should be a problem let the node image in this version

https://hub.docker.com/_/node